arroyo/processing/strategies/decoder.py

```diff
@@ -98,8 +98,11 @@ def validate(self, data: T) -> None:
 
 
 class JsonCodec(Codec[object]):
-    def __init__(self, json_schema: object) -> None:
-        self.__validate = fastjsonschema.compile(json_schema)
+    def __init__(self, json_schema: Optional[object] = None) -> None:
```

Comment:
I mean technically None is still an object. But I wanted to be very explicit that this is nullable.